cmake_modules/commonSetup.cmake

```diff
@@ -252,6 +253,24 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
 
   set(CMAKE_MODULE_PATH "${HPCC_SOURCE_DIR}/cmake_modules/")
 
+  if(UNIX AND SIGN_MODULES)
+    #export gpg public key used for signing to new installation
+    add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/pub.key
+      COMMAND gpg --export --output=${CMAKE_BINARY_DIR}/pub.key --batch --no-tty
+      BYPRODUCTS ${CMAKE_BINARY_DIR}/pub.key
```

Comment:
Alternatively move it down a line or two - that way older CMAKE's don't treat it as part of the command line (seems to just ignore it then).